.github/workflows/sync_theme.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - name: Checkout repo
-      uses: actions/checkout@v4
+      uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4
```

Comment:
so ugly :cry:. Does `dependabot` understand this?

Reply:
It is ugly but it is the recommended way :-/
Yes, dependabot not only updates it but also the commented out version after it.